examples/runFasterCode.ts

```diff
@@ -18,7 +18,9 @@ import * as mariner from '../src/mariner/index'; // This is used during developm
 import * as path from 'path';
 const config = mariner.readConfigFile('examples/config.json');
 
-const confluenceMarkdownPath = 'examples/confluenceMarkdown.md';
+const htmlPath = 'examples/output.html';
+// NOTE: Confluence output is disabled here by default, because HTML is usually preferred
```

Comment:
Rather than commenting it out should we just pull the confluence code completely?

Reply:
Usually I'm the one saying things like that. I agree. I'll remove it. I'm leaning toward renaming `runFasterCode` at the same time. To something like `runExample`.

Reply:
I removed the confluence calls from the example, and renamed the example to `runExample`. I left the confluence calls in place, in case anyone out there is using them, or if for some reason they would prefer that form of output over html.